stomp/__main__.py

```diff
@@ -20,9 +20,16 @@
                             Listen for messages on a queue/destination
   -V, --verbose             Verbose logging "on" or "off" (if on, full headers
                             from stomp server responses are printed)
-  --ssl                     Enable SSL connection
   --heartbeats=<heartbeats> Heartbeats to request when connecting with protocol >=
                             1.1 (two comma separated integers required) [default: 0,0]
+  --ssl                     Enable SSL connection
+  --ssl-key-file=<cert-file>
```

Comment:
@tcler This doesn't work with docopt (just tested it and it doesn't parse).  The variable names probably need to be unique.

Reply:
Oops, sorry will send new patch.